src/apps/routes/schemas/containers.clj

```diff
@@ -39,6 +39,7 @@
 (s/defschema Settings
   (describe
    {(s/optional-key :cpu_shares)         Integer
```

Comment:
Nope, we want to leave `cpu_shares` as an optional restriction admins can still set.
This PR is just removing the auto-restriction set on private tools added by users.